packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6258,6 +6258,9 @@ If the delete request is received for an active JSON Web Key, you receive an err
 }
 ```
 
+## Application OAuth 2.0 role assignment operations
+
+Please check [here](/docs/reference/api/roles/#role-assignment-operations) for OAuth 2.0 client application role assignment operations.
```

Comment:
We try to avoid using "here" and such for links.
```suggestion
See [Role assignment operations](/docs/reference/api/roles/#role-assignment-operations).
```